### DISCUSSION TITLE
Does evobot work with Slash Commands?

### DISCUSSION THREAD

**Discussion by St-Mich** [eritislami/evobot#1109] [Q&A] [answered]
I read recently that Discord will be forcing all bots to use Slash Commands (by April 2022). I was wondering if evobot had that feature built-in?

**Comment by St-Mich** [comment, marked as answer]
@eritislami

**Reply by eritislami** [reply]
No, unfortunately, that would require quite a bit of work to upgrade to v13 of discord.js and it's not possible for me to handle it right now, I am very busy with work and personal life
